### PR TITLE
Merging initial sass fixes upstream

### PR DESCRIPTION
**Pull request by spmcbride1201**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,9 +94,18 @@
...
+      "prettier --write",
+      "git add"
+    ],
+    "*.scss": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.less": [
+      "prettier --write",
+      "git add"
```
Comment (shanberg) [inline package.json]: We're still moving theoretical `less` files around even though there won't be any. Not a real issue, just a note.

File: src/style/components/nav-widget.scss [hunk 2]
```diff
@@ -48,13 +48,13 @@ $go-up-icon-size: $padding-base * 2;
...
   .nav-go-up,
   .nav-siblings {
-    @include mixin-button-style(
-      $nav-widget-text-color,
-      $nav-widget-background-color,
-      $nav-widget-background-color,
-      lighten
-    );
-    @include mixin-button-height($padding-base * 4);
+    // @include mixin-button-style(
```
Comment (shanberg) [inline src/style/components/nav-widget.scss]: I've commented out all the places I'm loading the button style builder. The button styles are in progress and will be fixed in a later PR.

File: src/style/components/routeBar.scss [hunk 3]
```diff
@@ -26,7 +26,7 @@ $active-route-bar-card-background: $decipher-green;
 
   // Conflicts with UK-Card styles.
   .route-bar-card-body {
-    @include uk-card-body();
+    // @include uk-card-body();
```
Comment (shanberg) [inline src/style/components/routeBar.scss]: I will need to fix the cards and other `less`-dependent imports from UIkit. Should not be a significant issue, just a separate PR.

Comment (spmcbride1201) [inline src/style/components/routeBar.scss]: We're pulling in the UIKit v3 scss source correctly, right? Is any of that stuff misconfigured?

**Pull request merged**